libs/etterlatte-brev-model/src/main/kotlin/no/nav/etterlatte/brev/Brevkoder.kt

```diff
@@ -40,6 +40,10 @@ enum class Brevkoder(
         EtterlatteBrevKode.OMSTILLINGSSTOENAD_INFORMASJON_DOEDSFALL,
         EtterlatteBrevKode.TOM_MAL_INFORMASJONSBREV,
     ),
+    OMS_INFORMASJON_MOTTATT_SOEKNAD(
```

Comment:
Denne brukes ikke av resten av koden, skal den tas i bruk senere?

Reply:
Godt spørsmål. Har egentlig bare tenkt at dette er en del av oppsettet for brev, men er kanskje ikke nødvendig.